example/sonivoxrender.c

```diff
@@ -241,20 +282,23 @@ int main (int argc, char **argv)
 {
     int ok = EXIT_SUCCESS;
     int index, c;
-    
+
     opterr = 0;
-    
-    while ((c = getopt (argc, argv, "hr:w:")) != -1) {
+
+    while ((c = getopt (argc, argv, "hd:r:w:")) != -1) {
         switch (c)
         {
         case 'h':
-            fprintf (stderr, "Usage: %s [-h] [-r 0..4] [-w 0..32765] file.mid ...\n"\
+            fprintf (stderr, "Usage: %s [-h] [-d file.dls] [-r 0..4] [-w 0..32765] file.mid ...\n"\
                         "Render standard MIDI files into raw PCM audio.\n"\
                         "Options:\n"\
                         "\t-h\tthis help message\n"\
```

Comment:
The help text should also mention the new program option:

```suggestion
                        "\t-h\tthis help message\n"\
                        "\t-d file.dls\tDLS soundfont\n"\
```

Reply:
Fix in the new commit.